OSDOCS-15852 [NETOBSERV] Modularize release notes 1.2 (#97627)
openshift/openshift-docs: modules/network-observability-release-notes-1-2-0-known-issues.adoc

// Module included in the following assemblies:
// * network_observability/network-observability-operator-release-notes-archive.adoc

:_mod-docs-content-type: REFERENCE
[id="network-observability-release-notes-1-2-0-known-issues_{context}"]
= Network observability 1.2.0 known issues

[role="_abstract"]

> Review the following issues and their recommended workarounds (where avaibale) that might affect your use of the Network Observability Operator 1.2.0 release.

Reviewer: ```suggestion
You can review the following issues and their workarounds, if available, to troubleshoot issues with the Network Observability Operator 1.2.0 release.
```
